docs/pipeline-configuration-files/specifications.mdx

```diff
@@ -276,22 +277,23 @@ beginning.
   identify the processor inside the pipeline. If multiple processors inside the
   pipeline use the same ID, provisioning will fail.
 
-### type
+### plugin
 
 - **Data Type**: String
 - **Required**: Yes
 - **Default**: None
-- **Description**: Defines the processor type (e.g. `js`). Check out the
-  [processors documentation](/docs/processors/getting-started) to find out which
-  processors are supported.
+- **Description**: Defines the processor's plugin name (e.g. `field.set`). Check out the
+  [processors documentation](/docs/processors/builtin) to find the list of builtin processors
+   we provide. Also, check [Building processors](/docs/processors/building-processors)
```

Comment:
note: this document is not created yet :D

Reply:
Maybe this could be the link to the page @lovromazgon will create on adding a page about WASM processors?

Reply:
Hmm, maybe not the same page, I'm not sure. I thought we'd have a page explaining how to build your own processor and a separate one explaining how WASM processors work underneath the hood (or would that be too in-depth for our users?).

Reply:
hmm we could have a page called "WASM in depth" or "processor's architecture" or sth like that, to talk about WASM underneath the hood, but I think this should be separate from the page about building your own processor

Reply:
I agree with @maha-hajja . We should keep "how to build" and "in depth/architecture" as separate pages. Maybe add a cross link in between at the bottom to provide additional context. 

Just to clarify, since It seems like we'll need to create an additional page either way, @lovromazgon what did you have in mind to tackle as part of ` Add a page about WASM processors` in https://github.com/ConduitIO/conduit/issues/1422? Would it be "How to build your own processor", maybe?